traffic_router/core/src/main/java/org/apache/traffic_control/traffic_router/core/dns/NameServer.java

```diff
@@ -192,7 +193,7 @@ private static void addAuthority(final Zone zone, final Message response, final
 		response.getHeader().setFlag(Flags.AA);
 	}
 
-	private static void addSOA(final Zone zone, final Message response, final int section, final int flags) {
+	private static void addSOA(final Zone zone, final Message response, final int section, final int flags, final long negativeCachingTTL) {
```

Comment:
Looks like the argument given for the `negativeCachingTTL` parameter is always `getNegativeCachingTTL()`. Why not make `negativeCachingTTL` static and use it directly instead without the function parameter?